google-services.json

```diff
@@ -0,0 +1,39 @@
+{
+  "project_info": {
+    "project_number": "795165933100",
+    "project_id": "altara-a24f7",
+    "storage_bucket": "altara-a24f7.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:795165933100:android:d9f615f27c391e7e3f9b89",
+        "android_client_info": {
+          "package_name": "com.altara.mobile"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "795165933100-t3budtlt6n53cf5v1be0buf5a6sda66b.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "AIzaSyBgewLXsCMLiVkiWGllEqig4VECT5fTaVo"
```

Comment:
can't we pass this as ENV?
Not good putting keys in code

Reply:
I can't do that here. It is a json file.